src/app/pages/utils/applyBasicPageHandlers.js

```diff
@@ -0,0 +1,22 @@
+import pipe from 'ramda/src/pipe';
+import withContexts from '#containers/PageHandlers/withContexts';
+import withPageWrapper from '#containers/PageHandlers/withPageWrapper';
+import withError from '#containers/PageHandlers/withError';
+import withLoading from '#containers/PageHandlers/withLoading';
+import withData from '#containers/PageHandlers/withData';
+import _withVariant from '#containers/PageHandlers/withVariant';
+
+export default ({ withVariant } = { withVariant: false }) =>
+  pipe(
+    withData,
+    withError,
+    withLoading,
+    withPageWrapper,
+    withContexts,
+    component => {
+      if (withVariant) {
+        return _withVariant(component);
+      }
+      return component;
+    },
```

Comment:
Few one-line options for this, not sure if any of them are clearer:

```suggestion
    withVariant ? _withVariant : component => component
```

```suggestion
    withVariant ? _withVariant : identity
```

```suggestion
    when(withVariant, _withVariant)
```

Reply:
I'm not sure I follow any of these any better than the original, I don't really understand the middle one tbh (that may be just me though!)

Anyone else got any thoughts? @jroebu14 ?

Reply:
I understand why it's something to pick on. It's one thing I wasn't happy about because it takes a minute to understand.

I was thinking of change the argument naming to `addVariantSupport` or `addVariantHandling` which might make any of the above, including the original clearer

e.g.

```js
    component => {
      if (addVariantHandling) {
        return withVariant(component);
      }
      return component;
    },
```

Reply:
I think @jroebu14's suggestion is a bit clearer personally, don't mind though